sidecar/sidecar.py

```diff
@@ -57,6 +68,15 @@ def watchForChanges(label, targetFolder, url, method, payload, current):
         print(f'Working on configmap {metadata.namespace}/{metadata.name}')
         if label in event['object'].metadata.labels.keys():
             print("Configmap with label found")
+
+            if folderAnnotation is not None:
+                if event['object'].metadata.annotations is not None:
+                    if folderAnnotation in event['object'].metadata.annotations.keys():
+                        destFolder = event['object'].metadata.annotations[folderAnnotation]
+                        print(f'Found a folder override annotation, placing the configmap in: {destFolder}')
+                    else:
+                        destFolder = targetFolder
```

Comment:
Seems a bit redundant to set `destFolder = targetFolder` here as well as on line 56.

Reply:
nice find, removed.